code-orange/Communicator.swift

```diff
@@ -64,3 +75,25 @@ extension Communicator: DataFetcher {
     return infectedLocations
   }
 }
+
+extension Communicator: DataSetter {
+  func shareInfectedLocations(_ locations: [COLocation]) {
+    guard let url = URL(string: serverUrl) else { return }
+    var json = [String: Codable]()
+    json["patientStatus"] = "carrier"
```

Comment:
Cool Renen, consider putting `carrier` and `country` in some variable so we can reuse them with the GET call

Reply:
I created a codable struct for it 🚀